internal/postgres/application_service.go

```diff
@@ -40,6 +40,7 @@ type dbApplication struct {
 	Is18OrOlder          sql.NullBool
 	MLHCodeOfConduct     sql.NullBool
 	MLHContestAndPrivacy sql.NullBool
+	CheckedInStatus      sql.NullBool
```

Comment:
This struct should reflect the database as well, but using sql variables. Basically makes it easier to read from db. So additional attributes should be reflected here as well.